_api-reference/analyze-apis/perform-text-analysis.md

```diff
@@ -635,7 +635,7 @@ PUT /books2
   }
 }
 ````
-The preceding request is an index API rather than an analyze API. See [DYNAMIC INDEX SETTINGS]({{site.url}}{{site.baseurl}}/opensearch/rest-api/index-apis/create-index/#dynamic-index-settings) for additional details.
+The preceding request is an index API rather than an analyze API. See [DYNAMIC INDEX SETTINGS]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#dynamic-index-settings) for additional details.
```

Comment:
Sentence case for the link text (the corresponding header should also be sentence case but it looks like the h4 or h5 is doing some kind of `toUpper()` conversion.